_data/issues/date-picker.yml

```diff
@@ -0,0 +1,19 @@
+title: Date picker
+type: component
+alert-type: info
+items:
+- date: 2023-09-29
+  summary: Users had trouble correctly typing dates in the input field.
+  summaryAdditional:
+  affectsAll: true
+  affectsScreenReaders: true
+  githubRepo: uswds
+  githubNumber: 5534
+- date: 2023-09-29
+  summary: JAWS user could not tab to the calendar icon with keyboard navigation.
+  summaryAdditional:
+  affectsJaws: true
+  affectsKeyboard: true
+  affectsScreenReaders: true
+  githubRepo: uswds
+  githubNumber: 5535
```

Comment:
Should we remove this item before testing? The finding was from only one user and it hasn't been confirmed by the a11y team as a JAWS bug yet